packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/go/changeEveryMapFile2.yml

```diff
@@ -0,0 +1,27 @@
+languageId: go
+command:
+  version: 6
+  spokenForm: change every map file
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - type: everyScope
+          scopeType: {type: map}
+        - type: containingScope
+          scopeType: {type: document}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: _ = []T{T{ /* foo */ /*0,*/ /* foo */ }, T{}}
```

Comment:
I'm a bit confused by this test. I was imagining you'd put my example in a file, say "cursorless record error", put your cursor next to the structure, and say "change map". If it's an error, then we're good

Reply:
haha. ok, will re-record something simpler next pass through my PRs.

Reply:
Oh yeah, way easier to read, thank you! I find myself quite convinced, and it's useful to know that's how `*` behaves